libproxy/cmake/pkgconfig.cmk

```diff
@@ -1,4 +1,4 @@
-if(NOT WIN32 AND NOT APPLE)
+if(NOT WIN32 AND NOT (APPLE AND WITH_APPLE_WEBKIT))
```

Comment:
The webkit condition should not matter at all here - find_package pkgconfig is valid with or without webkit explicitly being enabled

Reply:
So is just `if(NOT WIN32)` ok?